libraries/AP_Motors/AP_MotorsTri.cpp

```diff
@@ -309,7 +303,7 @@ void AP_MotorsTri::thrust_compensation(void)
 {
     if (_thrust_compensation_callback) {
         // convert 3 thrust values into an array indexed by motor number
-        float thrust[4] { _thrust_right, _thrust_left, 0, _thrust_rear };
+        float thrust[4] { _thrust_right, _thrust_left, 0.0f, _thrust_rear };
```

Comment:
I do not understand the meaning of "{" after whitespace necessary," ("after the white space is unnecessary, I do not think that neither is unnecessary.